registry.k8s.io/images/k8s-staging-kueue/images.yaml

```diff
@@ -1,5 +1,6 @@
 - name: kueue
   dmap:
+    "sha256:dc5a7ecd6daec5063979e2d43112ea43b4d829d6369541869f1f662980e86c0e": ["v0.8.1"]
```

Comment:
```shell
$ docker inspect us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.8.1 | grep dc5a7ecd6daec5063979e2d43112ea43b4d829d6369541869f1f662980e86c0e
        "Id": "sha256:dc5a7ecd6daec5063979e2d43112ea43b4d829d6369541869f1f662980e86c0e",
            "us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue@sha256:dc5a7ecd6daec5063979e2d43112ea43b4d829d6369541869f1f662980e86c0e"
```

Reply:
Is this still the way to graduate tho? Give that we are now using artifact registry?

Reply:
I guess it is? because of this configuration file https://github.com/borg-land/k8s.io/blob/6667ae641aa4ce7c62005574ce8434ee6b114121/registry.k8s.io/manifests/k8s-staging-kueue/promoter-manifest.yaml

Reply:
Also commit matches:

```
docker run --rm us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.8.1                                      
{"level":"info","ts":"2024-09-06T12:31:26.824891879Z","logger":"setup","caller":"kueue/main.go:123","msg":"Initializing","gitVersion":"v0.8.1","gitCommit":"7cb1dfb41e178aeb0674e36c996e17c711325e89"}
```